1.9/usage/managing-services/private-docker-registry.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: Using a Private Docker Registry 
+menu_order: 004.5
+---
+
```

Comment:
This topic needs an introduction. Right now it immediately jumps into the task without context. 
- where does the `docker.tar.gz` file come from?
- where does the `uris` field come from? what app? is this a marathon app definition parameter?